src/routes/Boards/Templates/Templates.tsx

```diff
@@ -184,37 +193,42 @@ export const Templates = () => {
         <section className="templates__container templates__container--recommended">
           {renderContainerHeader("left", t("Templates.recommendedTemplates"))}
           <div className="templates__card-container">
-            {fullRecommendedTemplates
-              .filter((rc) => matchSearchInput(rc.template))
-              .map((templateFull) => (
-                <TemplateCard
-                  templateType="RECOMMENDED"
-                  template={mergeTemplateWithColumns(templateFull.template, templateFull.columns)}
-                  onSelectTemplate={onSelectTemplateWithColumns}
-                  key={templateFull.template.id}
-                />
-              ))}
+            {sortBy(
+              fullRecommendedTemplates.filter((rc) => matchSearchInput(rc.template)),
+              (tc: TemplateWithColumns) => !recommendedFavourites.includes(tc.template.id)
+            ).map((templateFull: TemplateWithColumns) => (
+              <TemplateCard
+                templateType="RECOMMENDED"
+                template={{
+                  ...mergeTemplateWithColumns(templateFull.template, templateFull.columns),
+                  template: {
+                    ...templateFull.template,
+                    favourite: recommendedFavourites.includes(templateFull.template.id),
+                  },
+                }}
```

Comment:
instead of merging the favourite state here, one can also change the type of `ImportReducedTemplateWithColumns` by adding `favourite` (default `false`), and adjust `convertJsonToFullTemplates()` to update the state respectively. that would be a little cleaner I think